__app_test_mode/README.rst

```diff
@@ -12,9 +12,9 @@ This application stand-alone binaries for USB test modes. There are four build c
     - Test_K
     - Test_Packet
 
-The application enters it's respective test mode from boot, this removing the requirement to set the mode from a host via the  USBHSETT tool.
+The application enters it's respective test mode from boot, thus removing the requirement to set the mode from a host via the  USBHSETT tool.
 
-Note, you not shouild expect the the device to appear on any USB bus and its probabky not advisable to plug into any standard host. 
+Note, you not should expect the the device to appear on any USB bus and its probably not advisable to plug into any standard host. 
 
 This binaries are most commonly used in device characterisation.
```

Comment:
Surely "this binary" or "these binaries"?

Reply:
Fixed